src/components/button/button-theme.scss

```diff
@@ -5,6 +5,10 @@ md-toolbar .md-button.md-THEME_NAME-theme.md-fab {
   background-color: white;
 }
 
+button.md-no-style {
```

Comment:
Does this actually match anything? The element is a `<div>` in JS....I'm wondering if we can make it a real `button`. I got around that Safari layout bug by wrapping its contents in a `flex` element.